### DISCUSSION TITLE
Consolidate Keymaps in <leader>.... to leave some room for extensions

### DISCUSSION THREAD

**Discussion by appelgriebsch** [LazyVim/LazyVim#41] [Ideas]
With all the current default keymaps being added to `<leader>` space I really have some challenges to add additional top-level sections from my custom setup - e.g. I had `<leader>r` for Rust specific settings and `<leader>j` for Java.... both worked initially, but were picked up by LazyVim base recently...

When looking through the top-level items in the `<leader>` space you can notice that some keymaps have double entries (e.g. `<leader>e` for Nerdtree vs. `<leader>fe` for the same; also some top-level items just hold one entry (e.g. `<leader>o` just have an entry to open a terminal - ok, with 2 different cwd settings....)...

Maybe we can organize the functionality from base LazyVim a bit better, so that custom setups have some room to capture top-level entries as well?!

Happy to work / discuss some structure....

**Comment by folke** [comment]
You're totally right. Was already thinking about that too.

`<leader>j` is still there, but will move redraw somewhere else.

Same with some other mappings. Will think about it a bit more.

Feel free to post some suggestions :)

**Comment by folke** [comment]
I've just refactored a bunch of mappings to:
![image](https://user-images.githubusercontent.com/292349/211862747-06aa3760-69c7-485a-beb0-388342260d33.png)

`<leader>r,o,t,h,n` are now no longer used. 

I did add `<leader>u` (**UI**) though that now contains toggle and clear notifs

![image](https://user-images.githubusercontent.com/292349/211864971-33eab66e-b6bf-4b78-b5ab-56ad67e03d9e.png)

**Comment by luxus** [comment]
could this help with planing the keys?
https://docs.google.com/spreadsheets/d/1EJMLr_MPrYiO1TKJ2MjNkR-fA5Wgxa782-f0Wtdpz0w/edit#gid=1302333873

**Reply by ramgp** [reply]
Is there a reason `<leader>f` is not consistent, when the dashboard is displaying pressing `f` launches "Find Files (root dir)" once you are in a buffer in normal mode it takes `<leader>` + two taps to `f` to launch it again?

I know this might fall into the group of personal preferences, I guess I got used to the way these same bindings work in LunarVim

**Reply by folke** [reply]
The mappings on the dashboard are just shortcuts. On the dashboard you can type just `f`, no need to do `<leader>ff`.
